feature/interface/aggregate/otg_tests/aggregate_test/aggregate_test.go

```diff
@@ -218,7 +218,7 @@ func (tc *testCase) configureDUT(t *testing.T) {
 	fptest.LogYgot(t, "LACP", lacpPath, lacp)
 	lacpPath.Replace(t, lacp)
 
-	tc.dut.Telemetry().Interface(tc.aggID).OperStatus().Await(t, 20*time.Second, opUp)
+	time.Sleep(5 * time.Second)
```

Comment:
why blind sleep?

Reply:
Ideally we don't want anything here, but without the sleep we're seeing errors with some dut vendors on the next "Replace" _aggPath.Replace(t, agg)_